apps/homework/tasks.py

```diff
@@ -134,5 +134,7 @@ def post_submission(submission_pk, data_file=None):
         )
         logger.info(f'Succeeded posting submission with id {submission.id} to phase.')
     else:
+        logger.info(phase_final_resp.status_code)
```

Comment:
Extra little bug fix:
It's nice to see what Codalab responds with in the logs in the case of a failed submission POST.

Reply:
cool